test/models/site_test.rb

```diff
@@ -61,4 +61,35 @@ class SiteTest < ActiveSupport::TestCase
       assert_equal(["はすでに存在します"], site.errors[:fqdn])
     end
   end
+
+  sub_test_case "category_title_format" do
+    test "null" do
+      site = build(:site)
+      assert_true(site.valid?)
+    end
+
+    test "blank" do
```

Comment:
It is intentional that the assertion for "blank" is not included to the following `data`?